beekeeper.yml

```diff
@@ -6,8 +6,8 @@ pull_request:
                 name: Python lint checks
                 task: beefore
     - smoke-test:
-        task: py34
-        name: Smoke build (Python 3.4)
+        task: py35
+        name: Smoke build (Python 3.5)
     - full-test:
         subtasks:
             - py3.5:
```

Comment:
No need to re-run 3.5 if the smoke test passed.

Reply:
Ok, updated to not re-run the py35 test.